test/unit/translations/maps/indexMap.spec.js

```diff
@@ -0,0 +1,125 @@
+import { IndexMap } from 'handsontable/translations';
+
+it('should work with get, and set functions properly', () => {
+  const indexMap = new IndexMap();
+
+  indexMap.setValueAtIndex(0, 2);
+  indexMap.setValueAtIndex(1, 1);
+  indexMap.setValueAtIndex(2, 0);
+
+  expect(indexMap.getValues()).toEqual([]);
+  expect(indexMap.getLength()).toEqual(0);
+
+  indexMap.init(3);
+
+  expect(indexMap.getValues()).toEqual([0, 1, 2]);
+  expect(indexMap.getLength()).toEqual(3);
+
+  indexMap.setValueAtIndex(0, 2);
+  indexMap.setValueAtIndex(1, 1);
+  indexMap.setValueAtIndex(2, 0);
+
+  expect(indexMap.getValues()).toEqual([2, 1, 0]);
+  expect(indexMap.getLength()).toEqual(3);
+
+  indexMap.setValues([1, 2, 0]);
+
+  expect(indexMap.getValues()).toEqual([1, 2, 0]);
+  expect(indexMap.getLength()).toEqual(3);
+});
+
+it('should clear values properly', () => {
+  const indexMap = new IndexMap();
+
+  indexMap.init(3);
+  indexMap.setValues([1, 2, 0]);
+  indexMap.clear();
```

Comment:
I think that it's not necessary. It would duplicate some other checks. There is an extra test for `setValues` function in a complex way in test named: `should work with get, and set functions properly` for every single kind of map.